backend/test/course_test.js

```diff
@@ -24,7 +21,10 @@ describe("Test retrieve course by tag", function() {
       "getCourseByTagTest",
       "getCourseByTagTest",
       "getCourseByTagTest@email.com",
-      260561055
+      260561055,
```

Comment:
Put these as variables so that only 1 place needs to be modified in case something is changed